src/app/query/bitcoin/transaction/use-bitcoin-broadcast-transaction.ts

```diff
@@ -18,8 +19,21 @@ export function useBitcoinBroadcastTransaction() {
   const analytics = useAnalytics();
 
   const broadcastTx = useCallback(
-    async ({ tx, onSuccess, onError, onFinally, delayTime = 700 }: BroadcastCallbackArgs) => {
+    async ({
+      tx,
+      onSuccess,
+      onError,
+      onFinally,
+      delayTime = 700,
+      checkForInscribedUtxos,
```

Comment:
Perhaps a bypass option might make more sense like you did with filters?

```
skipCheckForInscribedUtxos = false
```

So it's on by default, and intentionally needs to be skipped?

Edit: just noticed its a callback. Will it need to change ever?

Reply:
I believe not